connector-webhook/src/main/resources/application.properties

```diff
@@ -1,13 +1,14 @@
+notifications.connector.http.follow-redirects=true
 notifications.connector.kafka.incoming.group-id=notifications-connector-webhook
 notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
 notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
 notifications.connector.name=webhook
 notifications.connector.redelivery.counter-name=camel.webhook.retry.counter
-notifications.connector.alternative.names=ansible
 # The following value matches the default value of the `connectionsPerRoute` option from the Camel `http` component.
 notifications.connector.seda.concurrent-consumers=20
 # The following value matches the default value of the `connectionsPerRoute` option from the Camel `http` component.
 notifications.connector.seda.queue-size=20
+notifications.connector.supported-connector-headers=${notifications.connector.name},ansible
```

Comment:
Shouldn't we have `webhook,ansible` on the `${notifications.connector.name}` parameter ?